### PR TITLE
SEO-188026-AngularJS-Dialog-appearance-and-styling-H1-Missing

### PR DESCRIPTION
**Pull request by Faith-Atieno**
Title | Description
-- | --
|Task Category | Site Audit|
|Content Task Link/Mail Screenshot | NA|
|Hotfix |https://github.com/syncfusion-content/angularjs-docs/pull/280|
|Ticket/Task link | https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/187689|
|Work link |https://syncfusion.sharepoint.com/:x:/r/sites/SEOWebsiteAuditing/_layouts/15/Doc.aspx?sourcedoc=%7B0C07BBB1-1233-47A7-818C-275A98FD0063%7D&file=Faith-%20Site%20Audit.xlsx&action=default&mobileredirect=true|

Changes Made | Reason for change |
|-- | -- |
|Changed-H1 Missing | To improve performance of the page |

### CODE REVIEW

**Comment by Faith-Atieno** [comment]
rebuild

File: angularjs/Dialog/appearance-and-styling.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-description: appearance and styling
-platform: AngularJS
+description: Learn here about appearance and styling with Syncfusion Essential Angular Dialog control, its elements, and more.
+platform: angularjs
 control: Angular- Dialog
 documentation: ug
 ---
 
-## Appearance and styling
+# Appearance and styling
```
Comment (Yvone-Atieno) [inline angularjs/Dialog/appearance-and-styling.md:10]: Include platform and control name

File: angularjs/Dialog/appearance-and-styling.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
 ### Showing header
```
Comment (Yvone-Atieno) [inline angularjs/Dialog/appearance-and-styling.md:12]: Change this as H2

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: angularjs/Dialog/appearance-and-styling.md (hunk 2 above)
Comment (Faith-Atieno) [reply angularjs/Dialog/appearance-and-styling.md]: resolved

File: angularjs/Dialog/appearance-and-styling.md (hunk 1 above)
Comment (Faith-Atieno) [reply angularjs/Dialog/appearance-and-styling.md]: resolved

File: angularjs/Dialog/appearance-and-styling.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
-description: appearance and styling
-platform: AngularJS
+description: Learn here about appearance and styling with Syncfusion Essential Angular Dialog control, its elements, and more.
+platform: angularjs
 control: Angular- Dialog
 documentation: ug
 ---
 
-## Appearance and styling
+# Appearance and styling in angularjs Dialog
```
Comment (Yvone-Atieno) [inline angularjs/Dialog/appearance-and-styling.md:10]: Check the platform name

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: angularjs/Dialog/appearance-and-styling.md [hunk 4]
```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
 title: Syncfusion Dialog appearance-and-styling
-description: appearance and styling
-platform: AngularJS
+description: Learn here about appearance and styling with Syncfusion Essential Angular Dialog control, its elements, and more.
```
Comment (Yvone-Atieno) [inline angularjs/Dialog/appearance-and-styling.md:4]: use in not with

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: angularjs/Dialog/appearance-and-styling.md [hunk 5]
```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
 title: Syncfusion Dialog appearance-and-styling
-description: appearance and styling
-platform: AngularJS
+description: Learn here about appearance and styling use in Syncfusion Essential Angular Dialog control, its elements, and more.
```
Comment (Yvone-Atieno) [inline angularjs/Dialog/appearance-and-styling.md:4]: Check description and title

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: angularjs/Dialog/appearance-and-styling.md (hunk 5 above)
Comment (Faith-Atieno) [reply angularjs/Dialog/appearance-and-styling.md]: resolved

File: angularjs/Dialog/appearance-and-styling.md [hunk 6]
```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
-title: Syncfusion Dialog appearance-and-styling
-description: appearance and styling
-platform: AngularJS
+title: Syncfusion Angular Dialog appearance-and-styling
+description: Learn here about appearance and styling support Syncfusion Angular Dialog control, its elements, and more.
```
Comment (Yvone-Atieno) [inline angularjs/Dialog/appearance-and-styling.md:4]: Check title and description

File: angularjs/Dialog/appearance-and-styling.md [hunk 7]
```diff
@@ -1,15 +1,15 @@
...
+platform: angularjs
 control: Angular- Dialog
 documentation: ug
 ---
 
-## Appearance and styling
+# Appearance and styling in Syncfusion Angular Dialog
```
Comment (Yvone-Atieno) [inline angularjs/Dialog/appearance-and-styling.md:10]: Remove the word syncfusion

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: angularjs/Dialog/appearance-and-styling.md (hunk 7 above)
Comment (Faith-Atieno) [reply angularjs/Dialog/appearance-and-styling.md]: resolved

File: angularjs/Dialog/appearance-and-styling.md (hunk 5 above)
Comment (Faith-Atieno) [reply angularjs/Dialog/appearance-and-styling.md]: resolved

File: angularjs/Dialog/appearance-and-styling.md (hunk 6 above)
Comment (Faith-Atieno) [reply angularjs/Dialog/appearance-and-styling.md]: resolved

File: angularjs/Dialog/appearance-and-styling.md [hunk 8]
```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
-title: Syncfusion Dialog appearance-and-styling
-description: appearance and styling
-platform: AngularJS
+title: Appearance and styling in Angular Dialog Component | Syncfusion
+description: Learn here all about Style and appearance in Syncfusion Angular Dialog control, its elements, and more.
```
[the remark this answers is not in the document]
Comment (Faith-Atieno) [reply angularjs/Dialog/appearance-and-styling.md]: resolved

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**